web/cobrands/greenwich/layout.scss

```diff
@@ -180,13 +265,43 @@ body.mappage {
     }
 
     .nav-menu {
-        a, span {
+        a,
+        span {
             color: $primary_text;
         }
         a:hover {
             text-decoration: underline;
         }
     }
+    // Around page
+    .pagination {
+        .next,
+        .prev {
+            color: $contrast_text;
+            border-radius: 0.25em;
+            &:hover {
+                text-decoration: underline;
+            }
+        }
+    }
+
+    .site-footer {
+        .container {
+            .site-footer__content {
+                .site-footer__logo {
+                    .identity {
+                        .identity__link {
+                            .icon-logo-white {
+                                display: none;
```

Comment:
Holy nest-ception batman! At this point, I‘d probably consider a `.site-footer .icon-logo-white { display: none !important; }` just to keep the complexity down.

But, to be honest, I think you can just `.site-footer { display: none; }` here, since the footer is hidden behind the map anyway.

(In fact, I wonder why we don’t do this by default in the core `_layout.scss` files. Any ideas, @dracos?